web/src/layout/map.jsx

```diff
@@ -85,8 +85,10 @@ export default () => {
     dispatchApplication('isLoadingMap', false);
 
     // Disable the map and redirect to zones ranking.
-    dispatchApplication('webGLSupported', false);
-    history.push({ pathname: '/ranking', search: location.search });
+    if (e.error === 'WebGL not supported') {
+      dispatchApplication('webGLSupported', false);
+      history.push({ pathname: '/ranking', search: location.search });
```

Comment:
This should probably have been changed outside this PR already but unless I'm mistaken there is no way to navigate away from the `/ranking` page on mobile.

Now here it actually works out in our favor somewhat since the map won't show if WebGL isn't supported but still, this should probably be addressed in a more elegant manner in another PR.

Reply:
Oh you do? 👀

I had some ideas around that and was thinking of doing some mockups for it that supported historical data and aggregates at Uni. (I am going to study Digital Design and Innovation (basically UX and graphic design)). But it would probably take a few weeks if not months.

Reply:
> Oh you do? 👀

Yes, we found that people are less inclined to use and instead navigate using the map.

> I had some ideas around that and was thinking of doing some mockups for it that supported historical data and aggregates at Uni. (I am going to study Digital Design and Innovation (basically UX and graphic design)). But it would probably take a few weeks if not months.

That sounds cool - would love to see some thoughts around it. We're still not completely settled on it, it is however a slightly complex component to keep alive because we have to consider it on every UI change as well as any larger data changes.